src/client/datascience/export/fileConverter.ts

```diff
@@ -55,14 +53,38 @@ export class FileConverter implements IFileConverter {
         sourceDocument: NotebookDocument,
         defaultFileName?: string,
         candidateInterpreter?: PythonEnvironment
+    ): Promise<undefined> {
+        const reporter = this.progressReporter.createProgressIndicator(
```

Comment:
Change here just hoisted the progress reporter for export up one level so that both import and export could use their own progress reporter but share the export implementation.